.github/workflows/tests.yml

```diff
@@ -27,11 +27,6 @@ jobs:
         restore-keys: ${{ runner.os }}-gradle
     - name: Run tests
       run: ./gradlew test --scan
-    - name: Run SonarQube scan
```

Comment:
Last bit of curiosity - why remove SonarQube?

Reply:
I no longer supports java 11. Info sec has added a ticket to our board to replace sonarqube with another tool